rh/apps.py

```diff
@@ -5,4 +5,4 @@ class RhConfig(AppConfig):
     name = "rh"
 
     def ready(self):
```

Comment:
since this just returns true all the time, maybe we should make an issue to delete this function altogether, unless it's required for some framework reason.

Reply:
Oh good point, I just removed it in 340dc1f1.